esp32-hal/examples/embassy_hello_world.rs

```diff
@@ -0,0 +1,60 @@
+#![no_std]
+#![no_main]
+#![feature(type_alias_impl_trait)]
+
+use embassy_executor::Executor;
+use embassy_time::{Duration, Timer};
+
+use esp32_hal::{
+    clock::ClockControl,
+    prelude::*,
+    timer::TimerGroup,
+    Rtc, embassy, pac::Peripherals,
+};
+use esp_backtrace as _;
+use static_cell::StaticCell;
+
+#[embassy_executor::task]
+async fn run1() {
+    loop {
+        esp_println::println!("Hello world from embassy using esp-hal-async!");
+        Timer::after(Duration::from_millis(10_000)).await;
```

Comment:
I think these durations are probably a bit excessive for the purpose of an example?